consensus/transactions.go

```diff
@@ -10,6 +10,7 @@ import (
 // Exported errors
 var (
 	ConflictingTransactionErr = errors.New("conflicting transaction exists in transaction pool")
+	InvalidSignatureErr       = errors.New("invalid signature in transaction")
```

Comment:
IIRC the stdlib convention is to use `Err` as the prefix, not suffix. But I think it's more readable this way.